buildModels.js

```diff
@@ -0,0 +1,29 @@
+var fs = require('fs');
```

Comment:
Yeah, in the viewer I feel like it's valid by association. XD gl-matrix happens to be the name of a package. There is no real consistency here. XD